src/index.html

```diff
@@ -2,11 +2,414 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
```

Comment:
Keep your attributes correctly formatted
![image](https://github.com/mate-academy/layout_catalog/assets/63243136/d4da7c25-7f56-4f54-9b33-abf0bd312db4)

Fix in all places